packages/kubernetes-1.27/Cargo.toml

```
@@ -14,8 +14,60 @@ path = "../packages.rs"
 package-name = "kubernetes-1.27"
 
 [[package.metadata.build-package.external-files]]
-url = "https://distro.eks.amazonaws.com/kubernetes-1-27/releases/34/artifacts/kubernetes/v1.27.14/kubernetes-src.tar.gz"
-sha512 = "c7a6acb127cfc510194b44a36d2f56883c89e363007312906e2aea5b21c53fd83b826f8661762e6b7afcab8027079c9d9bc1ba9f6848304845d16248698355b4"
+url = "https://dl.k8s.io/v1.27.16/kubernetes-src.tar.gz"
+sha512 = "fba641f745b5eef6f17e16501025959e570ae88912d79e2e08b4c79d18475430b879f98947d516871407feb297d5dac9b2fbc4ca565d3770c600d9550f3cffac"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/aws/eks-distro/blob/main/projects/kubernetes/kubernetes/1-27/patches/0001-EKS-PATCH-admission-webhook-exclusion-from-file.patch"
```

Need to use the raw URLs, otherwise the hashes won't match when retrieved via `curl`:
```suggestion
url = "https://raw.githubusercontent.com/aws/eks-distro/refs/heads/main/projects/kubernetes/kubernetes/1-27/patches/0001-EKS-PATCH-admission-webhook-exclusion-from-file.patch"
```